cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
```

Comment:
Thing is, you want to capture every line from the one you have found, until syncthing exits, because that's the full backtrace of the crash. And then ideally the last few lines leading up to that, for context.